src/css/profile/wearable/changeable/common/checkboxradio.less

```diff
@@ -98,10 +98,16 @@ input[type="checkbox"]:not(.ui-switch-input) {
 }
 
 .ui-checkbox-small {
+	font-size: 32 * @px_base;
+	color: #ffffff;
```

Comment:
Is there any variable for the #ffffff?
You can use the @_white instead of the '#ffffff'.

Reply:
In wearable profile, we have @white-color defined in theme-circle/theme.colorPalette.less.
Howver, .ui-checkbox-small is defined in common/checkboxradio.less where @white-color is not defined.
I can import theme-circle/theme.colorPalette.less in common/checkboxradio.less but I am not sure if this does not break something else, for examples, styles for square device.

Reply:
I see.
IMO, it seems necessary to arrange the parts related to color definition :(.